AdminWebsite/AdminWebsite/ClientApp/src/app/services/clients/api-client.ts

```diff
@@ -2794,6 +2851,45 @@ export interface ICaseResponse {
     is_lead_case?: boolean;
 }
 
+export class LinkedParticipantResponse implements ILinkedParticipantResponse {
```

Comment:
Same as above, this class isn't needed in TS

Reply:
this file (api-client.ts) is auto generated